.eslintrc.js

```diff
@@ -76,6 +76,8 @@ module.exports = {
 		globalThis: true,
 		window: true,
 		document: true,
+		File: true,
```

Comment:
This has been needed to solve some `jsdoc/no-undefined-types` errors in `client/lib/wpcom-undocumented/lib/undocumented.js`. cc @Automattic/team-calypso @sirreal

Reply:
I think I'd rather alter the code to use `globalThis.File` than `File` directly. @sgomes thoughts?

Reply:
This is a matter of codestyle, mostly. My personal opinion is that `File` isn't any more or less important than other global classes (like `URL` or `Blob`), so there's no reason why it should be promoted to an "acceptable" global, above others. And in the interest of keeping potential name collisions and unexpected bugs down, the current approach, while more verbose, is more reliable. So yes, I'd vote to alter the code to use `globalThis.File` as well :)

Do note that if you're going to be using `File` a bunch of times in a module/class/method, you could always create a constant and reuse it, instead of using `globalThis.File` everywhere:

```
const { File } = globalThis;
```

Reply:
Thanks! Done in 5d434f3. I admit I find kinda weird using `globalThis.File` in the JSDoc param types, but maybe it's just I'm not used to that syntax 🙂 .